src/Stratis.Bitcoin.IntegrationTests/APITests.cs

```diff
@@ -1,140 +0,0 @@
-﻿using System;
-using System.Net.Http;
-using System.Net.Http.Headers;
-using System.Text;
-using NBitcoin;
-using Stratis.Bitcoin.Features.Api;
-using Stratis.Bitcoin.Features.Miner;
-using Stratis.Bitcoin.Features.Miner.Models;
-using Stratis.Bitcoin.Features.Wallet;
-using Stratis.Bitcoin.Features.Wallet.Interfaces;
-using Stratis.Bitcoin.IntegrationTests.EnvironmentMockUpHelpers;
-using Xunit;
-
-namespace Stratis.Bitcoin.IntegrationTests.Api
-{
-    public class APITests : IDisposable
-    {
-        private bool initialBlockSignature;
-        private bool initialTimeStamp;
-
-        public APITests()
-        {
-            this.initialBlockSignature = Transaction.TimeStamp;
```

Comment:
Please add this to the `BeforeTest()` method so that we can stop a possible static flag issue.

Reply:
same as below